tutorials/html-syntax.md

```diff
@@ -0,0 +1,72 @@
+HTML syntax tutorial for Learn OCaml tutorials
+==============================================
+
+Main HTML structure
+-------------------
+
+Tutorials written in the HTML syntax are valid HTML files that can be
+viewed in any Web browser (but not all HTML files are valid tutorials).
+
+Here is an example HTML 5 stub that you can copy and past.
+
+    <html>
+      <head>
+         <meta charset='UTF-8'>
+         <title>This title is ignored.</title>
+         <!-- Anything here is ignored, you can add some
```

Comment:
Does that mean that this part is skip when include inside learn-ocaml? If so, make this fact explicit in the documentation.

Reply:
Actually I just copy/paste the basic tutorial directory from the demo because it is needed to build learn-ocaml in the current version ... sorry if my  commit messages were not clear about that. I have already report this issue.